### DISCUSSION TITLE
preVABS vs. iVABS

### DISCUSSION THREAD

**Discussion by dana-frye** [wenbinyugroup/.github#8] [VABS Related Discussions] [answered]
Hello All, 

I have a preVABS question - if I just want to use preVABS as a preprocessor for VABS, and I don't need to use any of the iVABS functionality to do any parametric optimization, how should I go about doing that? Is there a workflow to utilize the preVABS preprocessing functionality within iVABS? Or should I use a standalone preVABS installation?

Thanks in advance

**Comment by unitasium** [comment]
Can you provide a specific case of what you want to do? You can use PreVABS alone. It is a tool to create the geometry and mesh of a single cross-section for VABS. All inputs for defining the cross-section and configuring the analysis should be given in the xml input file. Then you call PreVABS in the command line to generate the cross-section and write VABS input file. For more details: https://wenbinyugroup.github.io/ivabs/ref/prevabs/contents/run.html

**Reply by dana-frye** [reply]
I understand all of that, I am wondering if I can do that (call PreVABS in command line) with the iVABS installation, or if I should use the standalone PreVABS installation)

**Reply by unitasium** [reply, marked as answer]
You can do either. iVABS does not overwrite/block the PreVABS command line interface.
